archai/nlp/datasets/lm_iterators.py

```diff
@@ -34,8 +35,8 @@ def __init__(self, input_ids, bsz, bptt, device='cpu', mem_len=None, ext_len=Non
             self.input_ids = torch.cat((warmup_ids, self.input_ids), dim=-1)
 
         # Partition input_ids for DistributedDataParallel
-        world_size = nv_utils.distributed.get_world_size()
-        rank = nv_utils.distributed.get_rank()
+        world_size = distributed.get_world_size()

```

Comment:
In general, it is good to use namespace prefix is sub-namespace is too general. For example, "distributed" name space also appears in PyTorch so attaching nv_utils makes it clear to reader where it is coming from.

Reply:
Yes, we need to get rid of nv_. So probably just use original prefix?

Reply:
Ok! Sounds great! I will change it then.